polaris/polaris/sep24/serializers.py

```diff
@@ -20,9 +23,18 @@ def get_more_info_url(self, transaction_instance):
         if not request_from_context:
             raise ValueError("Unable to construct url for transaction.")
 
-        path = reverse("more_info")
-        path_params = f"{path}?id={transaction_instance.id}"
-        return request_from_context.build_absolute_uri(path_params)
+        if "sep6" in self.context.get("request").build_absolute_uri():
+            path = None
+            if "sass_processor" in settings.INSTALLED_APPS:
```

Comment:
What does having sass installed have to do with the logic here?

Reply:
Polaris the UI pages require that package. SEP-6 doesn't require any UI pages, but `more_info_url` is optional. So if they want `more_info_url`, they have to have this installed, which they might not.

Thinking about this more I'd like to remove all the SASS stuff. We can compile all the SASS files offline before each release and reference the CSS files in the templates. This will simply the configuration too.

Filing an issue for it. For now, the docs describe when a Polaris user would need to have this installed.

Reply:
Just add a comment here explaining what the if statement implies, and then we have a task to remove sass altogether